### DISCUSSION TITLE
Possible to place logo above/below text?

### DISCUSSION THREAD

**Discussion by IsacDiamant** [kovidgoyal/kitty#6984] [Q&A] [answered]
From what I've seen, the logo is bound by the padding. Is it possible to circumvent this currently?

**Comment by kovidgoyal** [comment, marked as answer]
No.

On Tue, Jan 09, 2024 at 08:12:48AM -0800, Isac Diamant wrote:
> From what I've seen, the logo is bound by the padding. Is it possible to circumvent this currently?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6984
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
